decisions/DR-update-devkit-architecture.md

```diff
@@ -0,0 +1,63 @@
+# Update devkit architecture
+
+| Status      | draft |
+| :---------- | :-------------------------------------------------------------------------------------- |
+| **PR #**    | []() |
+| **Author & Sponsor** | [Marko](https://github.com/m4rk055) (marko@blindnet.io) |
+| **Reviewers** | [Milan](https://github.com/milstan), [Amaury](https://github.com/TheKinrar), [Justin](https://github.com/jboileau99) |
+| **Assignee** | [Milan](https://github.com/milstan) |
+
+## Context and Problem Statement
+
+The model described in the [PRIV](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md) document requires the [Privacy Compiler](https://github.com/blindnet-io/product-management/tree/main/refs/high-level-architecture#privacy-compiler) to be aware of every [Data Captures and Capture Fragments](https://github.com/blindnet-io/product-management/tree/main/refs/high-level-architecture#privacy-compiler) in the system.
+That puts a burden on the implementing system to
+- make sure every data point collected by the system is as well referenced (metadata) in the Privacy Computation Engine database
+- collect or transform data to a particular format (capture and fragments)
+
+This DR proposes removing the notion of Data Captures and Capture Fragments from Privacy Computation Engine.
```

Comment:
I like the idea of removing it from PCE but not necessairly in general. That way we make the components operate with minimal information which is sane.

Reply:
It stays in the Capture Component, removing it just from the PCE.